crafters/image/ImageNormalizer/tests/test_imagenormalizer.py

```diff
@@ -13,3 +13,10 @@ def test_transform_results():
     img_array = create_random_img_array(img_size, img_size)
     crafted_doc = crafter.craft(img_array)
     assert crafted_doc["blob"].shape == (224, 224, 3)
+
+    img_size = (217, 200)
+    target_size = (224, 224)
```

Comment:
And a test to have different width and height? That would be the point actually

Reply:
Giving one side only resizes short while giving tuple sizes both. The input has different h, w and so it has to change both.
I will push your change though.